src/mp-instance.ts

```diff
@@ -1267,6 +1269,31 @@ export default function mParticleInstance(this: IMParticleWebSDKInstance, instan
         }
     };
     // Used by our forwarders
+    this.onKitReady = function(moduleId, callback) {
+        const kitListener = self._preInit.kitListeners[moduleId];
+
+        if (!kitListener) {
+            self.Logger.error('Kit listener not found for moduleId: ' + moduleId);
+            return;
+        }
```

Comment:
I would DRY this with a method so that it's a helper function

Reply:
I think that work should be done as part of https://go.mparticle.com/work/SQDSDKS-4475